internal/app/machined/pkg/runtime/v1alpha1/platform/hcloud/hcloud.go

```diff
@@ -181,9 +156,14 @@ func (h *Hcloud) KernelArgs() procfs.Parameters {
 }
 
 // NetworkConfiguration implements the runtime.Platform interface.
-//
-//nolint:gocyclo
 func (h *Hcloud) NetworkConfiguration(ctx context.Context, _ state.State, ch chan<- *runtime.PlatformNetworkConfig) error {
+	log.Printf("fetching hcloud instance metadata from")
```

Comment:
```suggestion
	log.Printf("fetching hcloud instance metadata from %s", ??)
```

Reply:
removed, actually `getMetadata` prints the info